### DISCUSSION TITLE
Help about my own model trained data

### DISCUSSION THREAD

**Discussion by Navy1989** [julienkay/MobileNeRF-Unity-Viewer#8] [Q&A] [answered]
update:
it worked on my own data, thanks for your tool again, please detele this issule.

**Comment by julienkay** [comment, marked as answer]
Thanks for the update. Glad you got it working!

EDIT: By the way, feel free to leave questions like these up for others to see, even if it the solution seems trivial. Chances are, others run into the same issue and might get value from your post.

Also, the scene looked really cool from the screenshots you had posted before. Did you use [this code](https://github.com/google-research/jax3d/tree/main/jax3d/projects/mobilenerf) to train your own data?
